cwt/draft-ietf-ace-cbor-web-token.xml

```diff
@@ -384,22 +384,19 @@
             </list>
           </t>
           <t>
-              If a nested signing, MACing or encryption operation will be
-              performed, let the Message be the COSE_Sign/COSE_Sign1,
-              COSE_Mac/COSE_Mac0 or COSE_Encrypt/COSE_Encrypt0, and
-              return to Step 3, using a "content type" header value
-              corresponding to the media type "application/cwt" in the new
-              COSE Header created in that step.
-            <vspace/>
-              Note: If integrity (signing/MACing) and confidentiality
-              (encryption) protection are needed, it is recommended to use an
-              authenticated encryption algorithm to save space and processing.
+            Add the corresponding COSE CBOR tag to the output.
           </t>
           <t>
-	    If needed by the application, add the appropriate COSE CBOR tag
-	    to the COSE object to indicate type of COSE object.
-	    If also needed by the application, add the CWT CBOR tag
-	    to indicate that the COSE object is a CWT.
+            If a nested signing, MACing or encryption operation will be
+            performed, let the Message be the COSE_Sign/COSE_Sign1,
+            COSE_Mac/COSE_Mac0 or COSE_Encrypt/COSE_Encrypt0, and
+            return to Step 3, using a "content type" header value
+            corresponding to the media type "application/cwt" in the new
+            COSE Header created in that step.
+          </t>
+          <t>
+            If application layer cannot indicate that its content is a CWT add
+            the CWT CBOR tag.
```

Comment:
No it does not leave out the COSE CBOR tag, I have moved that part up to before the nesting options. 
The COSE CBOR tag needs to be added before nesting because 'content type' in the outer layer will say CWT so the inner layer MUST have a tag to indicate type of COSE message. But the CWT tag will not be used in the inner layer of the say reason that the outer layer will indicate that it contains a CWT.